common/component/redis/redis.go

```diff
@@ -22,8 +22,12 @@ import (
 
 	"golang.org/x/mod/semver"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
```

Comment:
```suggestion
	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
	"golang.org/x/mod/semver"
```

Reply:
I'ts just what our linter wants :) Ideally dependencies should be in 3 "blocks":

1. standard library
2. 3rd-party deps
3. Deps from "github.com/dapr"

Reply:
oh ya a good way of making it clean